sklearn/model_selection/tests/test_search.py

```diff
@@ -929,6 +961,109 @@ def test_search_iid_param():
         assert_almost_equal(train_std, 0)
 
 
+def test_grid_search_cv_results_multimetric():
+    X, y = make_classification(n_samples=50, n_features=4, random_state=42)
+
+    n_splits = 3
+    params = [dict(kernel=['rbf', ], C=[1, 10], gamma=[0.1, 1]),
+              dict(kernel=['poly', ], degree=[1, 2])]
+
+    for iid in (False, True):
+        grid_searches = []
+        for scoring in ({'accuracy': make_scorer(accuracy_score),
+                         'recall': make_scorer(recall_score)},
+                        'accuracy', 'recall'):
+            grid_search = GridSearchCV(SVC(), cv=n_splits, iid=iid,
+                                       param_grid=params, scoring=scoring,
+                                       refit=False)
+            grid_search.fit(X, y)
+            assert_equal(grid_search.iid, iid)
+            grid_searches.append(grid_search)
+
+        compare_cv_results_multimetric_with_single_metric_accuracy_recall(
+            *grid_searches, iid=iid)
+
+
+def test_random_search_cv_results_multimetric():
```

Comment:
I would perhaps prefer folding this into the grid search test. It's just getting too verbose and hard to see what differs.